### PR TITLE
Create SetLoadoutShuffleEnabled.md

### PR DESCRIPTION
**Pull request by GRUmbaWasHere**

### CODE REVIEW

File: Fortnite/MCP/operations/SetLoadoutShuffleEnabled.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# SetLoadoutShuffleEnabled
+**Description**: `Enables Loadout Shuffle` \
+**Profile**: `athena`
+
+## Body
+
+```js
+{
+    "loadoutType": "CosmeticLoadout:LoadoutSchema_Jam", // The Loadout Schema you want to use (see formating in the example!)
+    "bEnabled": false // false or true
```
Comment (LeleDerGrasshalmi) [inline Fortnite/MCP/operations/SetLoadoutShuffleEnabled.md:10]: Just say thats its a boolean
```
"bEnabled": false // Boolean
```

File: Fortnite/MCP/operations/SetLoadoutShuffleEnabled.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# SetLoadoutShuffleEnabled
+**Description**: `Enables Loadout Shuffle` \
+**Profile**: `athena`
+
+## Body
+
+```js
+{
+    "loadoutType": "CosmeticLoadout:LoadoutSchema_Jam", // The Loadout Schema you want to use (see formating in the example!)
```
Comment (LeleDerGrasshalmi) [inline Fortnite/MCP/operations/SetLoadoutShuffleEnabled.md:9]: just leave the string empty, and reference the `PutModularCosmeticLoadout` operation which has all possible loadout types
```
"loadoutType": ":LoadoutSchema_Jam", // The Loadout Section, see 'PutModularCosmeticLoadout' operation for possible values
```

File: Fortnite/MCP/operations/SetLoadoutShuffleEnabled.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# SetLoadoutShuffleEnabled
+**Description**: `Enables Loadout Shuffle` \
```
Comment (LeleDerGrasshalmi) [inline Fortnite/MCP/operations/SetLoadoutShuffleEnabled.md:2]: say that it is for a specific section
```
Enables shuffle for the specified loadout section.
```

**Review by LeleDerGrasshalmi — CHANGES_REQUESTED** [review]
see comments

**Review by GRUmbaWasHere — COMMENTED** [review]
👍

**Review by LeleDerGrasshalmi — APPROVED** [review]
lgtm

**Pull request merged**